rdkit/Chem/Features/ShowFeats.py

```diff
@@ -278,6 +278,8 @@ def ShowMolFeats(mol, factory, viewer, radius=0.5, confId=-1, showOnly=True, nam
   try:
     v = MolViewer()
   except Exception:
+    import traceback
```

Comment:
You can do the same thing with logger.exception which reduces some code.  The nice thing is that it follows the logging levels.

Reply:
```
logger.exception(
      'Unable to connect to PyMol server.\nPlease run ~landrgr1/extern/PyMol/launch.sh to start it.'
    )
```

Also, maybe remove the ~landrgr1 or change the message entirely